did.go

```diff
@@ -126,6 +195,52 @@ var DIDMethodNetwork = map[DIDMethod]map[DIDNetworkFlag]byte{
 	},
 }
 
+// DIDMethodNetworkOpts is a structure to represent DID method network options
+type DIDMethodNetworkOpts struct {
+	Method      DIDMethod
+	MethodByte  *byte
+	Blockchain  Blockchain
+	Network     NetworkID
+	NetworkFlag byte
+	ChainID     *int
+}
+
+// RegisterDIDMethodNetwork registers new DID method network
+func RegisterDIDMethodNetwork(opts DIDMethodNetworkOpts) error {
+
+	b := opts.Blockchain
+	n := opts.Network
+	m := opts.Method
+	blockchains[b] = b
+	networks[n] = n
+
+	err := RegisterDIDMethodWithByte(m, opts.MethodByte)
```

Comment:
do not call do not call RegisterDIDMethodWithByte id if option for chain id is not provided.
Make param mandatory